packages/contracts/contracts/minter-suite/Minters/MinterSetPriceERC20V5.sol

```diff
@@ -528,6 +524,11 @@ contract MinterSetPriceERC20V5 is ReentrancyGuard, ISharedMinterV0 {
             _coreContract,
             _isEngineCaches[_coreContract]
         );
+        // process payment in ERC20
```

Comment:
moved for consistency across minters, the change being driven by avoiding stack to deep error on polyptych minters